pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -209,10 +202,8 @@ func (r *ReconcileHyperConverged) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, err
```

Comment:
You should probably only requeue when the error is something other than it already exists.

Reply:
Now I see that `manageComponentResource` already does that. :+1: